src/components/LoadMoreButton.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <UiButton class="w-full">
+    {{ $t('Load more') }}
```

Comment:
This should use a key and be translated here https://github.com/snapshot-labs/snapshot/blob/develop/src/locales/default.json
```suggestion
    {{ $t('loadMore') }}
```